### PR TITLE
#58 - Fix Drupal deprecations

### PR DESCRIPTION
**Pull request by benjamintoussaint**

### CODE REVIEW

File: wmcontent.info.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: wmcontent
 type: module
 description: Adds configurable entity containers to entity types (for e.g. paragraphs)
 version: 2.2.0
-core_version_requirement: ^10
+core_version_requirement: ">=10"
```
Comment (RobinHoutevelts) [inline wmcontent.info.yml:5]: Doesn't this mean we also support Drupal 11 _the moment_ it gets released?
Shouldn't we keep `^10` and when Drupal 11 is available and this module has been tested, change it to `^10 || ^11`?

Comment (benjamintoussaint) [reply wmcontent.info.yml:5]: @RobinHoutevelts You're right, I might have misinterpreted these warnings 🤔 

```
modules/contrib/wmcontent/wmcontent.info.yml:
┌──────────┬──────┬──────────────────────────────────────────────────────────┐
│  STATUS  │ LINE │                         MESSAGE                          │
├──────────┼──────┼──────────────────────────────────────────────────────────┤
│ Check    │ 0    │ Value of core_version_requirement: ^10 is not compatible │
│ manually │      │ with the next major version of Drupal core. See          │
│          │      │ https://drupal.org/node/3070687.                         │
│          │      │                                                          │
└──────────┴──────┴──────────────────────────────────────────────────────────┘

modules/contrib/wmcontent/composer.json:
┌──────────┬──────┬──────────────────────────────────────────────────────────────┐
│  STATUS  │ LINE │                           MESSAGE                            │
├──────────┼──────┼──────────────────────────────────────────────────────────────┤
│ Check    │ 0    │ The drupal/core requirement is not compatible with the next  │
│ manually │      │ major version of Drupal. Either remove it or update it to be │
│          │      │ compatible. See                                              │
│          │      │ https://drupal.org/node/2514612#s-drupal-9-compatibility.    │
│          │      │                                                              │
└──────────┴──────┴──────────────────────────────────────────────────────────────┘
```

Comment (RobinHoutevelts) [reply wmcontent.info.yml:5]: Ah yeah, the upgrade_status module is already aiming for Drupal 11.
But I think it's too soon to act on those yet.

Drupal 10.3 is set to be released soonish and will have a lot more deprecations.

Comment (benjamintoussaint) [reply wmcontent.info.yml]: I've removed Drupal version requirement change 👍

**Review by RobinHoutevelts — CHANGES_REQUESTED** [review]
[The commit](https://git.drupalcode.org/project/drupal/-/commit/f9bf0787b9279c7690d27524e767ea180c1ba5aa) that adds `\Drupal\Core\Utility\Error::logException()` is only present in Drupal `>=10.1`.
I think we need to update `composer.json` to make it clear this module is only compatible with `"drupal/core": "^10.1"`.

**Review by RobinHoutevelts — APPROVED** [review]

**Pull request merged**